schematron/v2.1.0/L100_Audit.sch

```diff
@@ -0,0 +1,150 @@
+<?xml version='1.0' encoding='ASCII'?>
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+  <sch:ns prefix="auc" uri="http://buildingsync.net/schemas/bedes-auc/2019"/>
+  <sch:phase id="facility_description" see="ASHRAE 211 6.1.1">
```

Comment:
don't we also need to check document structure here?

Reply:
Need to know there is:
- Atleast 1 building
- Atleast 1 report, including atleast 4 scenarios (benchmark, target, measured, N+ low / no cost measures)
- Atleast 2 contacts